css/Dockerfile

```diff
@@ -4,18 +4,22 @@ FROM steamcmd
 USER root
 RUN dpkg --add-architecture i386; apt-get update;  apt-get install -y libstdc++6:i386 libcurl4-gnutls-dev:i386
 
-USER steam
-RUN mkdir -p /steam/css
+ENV GAME=css
+ENV APP_ID=232330
+
+RUN mkdir -p /steam/${GAME}/.cache/ \
+  && touch /steam/${GAME}/.cache/empty
+WORKDIR /steam/${GAME}/
+COPY .cache .
 
-WORKDIR /steam/steamcmd_linux
-RUN ./steamcmd.sh +login anonymous +force_install_dir ../css +app_update 232330 +quit
+RUN chown -R steam /steam/${GAME}/ \
+  && mv .cache/* . \
+  && rm -fr .cache
```

Comment:
Having a multi line RUN command will only create one layer. This is a bit of a moot point though as the caching "solution" I've added turns out to be quite slow locally.

I'm thinking that mounting a volume and running steamcmd update on the ENTRYPOINT rather than in the build will reduce image size, build time, and data usage. Especially with subsequent builds that change the underlying image layers.

Reply:
If you're meaning that you'd keep the CSGO or whatever game files outside of the container and mount them in using a volume, that's not gonna fly here. The game files must be inside the image, and must be installed/updated during image build - anything else isn't really 

Perhaps a solution that would work well for you - you can use a regular LANCache type thing so that the game install goes super quick. No need for complicated or hacky solutions here - they'll still install super fast. You could even have the cache run on your local docker installation if you wanted to avoid the cost of going over gigabit network. The steamcmd updates support using steamcache no problems.